### ISSUE TITLE
Application not exiting when closed

### ISSUE THREAD

**Issue by RupertAvery** [chromelyapps/Chromely#94]
- Using the latest code from master branch.
- Using the Vue demo app.
- Directly referencing Chromely.CefSharp.WinApi and Chromely.Core projects in my project.

Creating the window like so:

```csharp
                ChromelyConfiguration config = ChromelyConfiguration
                                              .Create(container)
                                              .WithAppArgs(args)
                                              .WithHostSize(1200, 900)
                                              .WithHostMode(WindowState.Normal, true)
                                              .WithHostTitle("delphi")
                                              .WithHostIconFile("chromely.ico")
                                              .WithStartUrl(startUrl)
                                              .WithLogFile("logs\\chromely.cef_new.log")
                                              .WithLogSeverity(LogSeverity.Info)
                                              .UseDefaultLogger()
                                              .UseDefaultResourceSchemeHandler("local", string.Empty)
                                              .UseDefaultHttpSchemeHandler("http", "chromely.com")
                                              .UseDefaultJsHandler("boundControllerAsync", true);


                using (var window = ChromelyWindow.Create(config))
                {
                    window.RegisterUrlScheme(new UrlScheme("https://github.com/mattkol/Chromely", true));
                    // 1. Register current/local assembly:
                    window.RegisterServiceAssembly(Assembly.GetExecutingAssembly());
                    window.ScanAssemblies();

                    return window.Run(args);
                }
```

Application does not exit when I press the Close (X) button on the top right of the window.

**Comment by RupertAvery** [comment]
Towards the end of the screencap, I close the windows by pressing X, the window closes but the application continues to run.

![chromely](https://user-images.githubusercontent.com/1910659/57529505-aa730080-7367-11e9-9730-4c3d2dda31d7.gif)

**Comment by mattkol** [comment]
@RupertAvery please see https://github.com/chromelyapps/Chromely/issues/93#issuecomment-491525455

**Comment by RupertAvery** [comment]
This seems to have been fixed by commit 44f8abee7053e3b255e3dda8b040bb23fd07ee47, thanks!
